### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by Sakrecoer**
Initial draft for the GitHub organization landing page.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+![Alephium Logo](https://github.com/alephium/alephium-brand-guide/blob/master/logos/grey/Logo-Horizontal-Grey.png "Alephium Logo")
```
Comment (polarker) [inline profile/README.md]: Let's use theme-aware logos like Hongchao? https://github.com/alephium/extension-wallet/blob/master/Readme.md?plain=1#L3

If we cannot use theme-aware logs, why do we expect the external services to use them?

**Comment by polarker** [comment]
I am a bit worried about the length of the text, not sure if it will fit well as an organization description. But we could try and see.

File: profile/README.md (hunk 1 above)
Comment (Sakrecoer) [reply profile/README.md]: Wow! This is great! Fixed in https://github.com/alephium/.github/commit/79d5646f2f4db327c6bdb6718e2d2b87ae851656

**Comment by Sakrecoer** [comment]
It looks ok. Maybe we only need the boiler plate and everything in "Start Here" to make it shorter? WDYT @MaudSimon ?

**Comment by MaudSimon** [comment]
Perfect!

**Pull request merged**